docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,145 @@
+
+| flag | envvar | example |
+| --- | --- | --- |
+| `--db-url` | `DATABASE_URL` | postgres://localhost/horizon_testnet |
+| `--network` | `NETWORK` | pubnet |
+| `--log-level` | `LOG_LEVEL` | info |
+| `--stellar-core-url` | STELLAR_CORE_URL | http://localhost:11626 |
+| `--tx-sub-enabled` | `TX_SUB_ENABLED` | true |
+
+## Notes
+
+- If you have configured your deployment to perform the ingestion role, then it is **strongly** recommended to review [Ingestion](./ingestion.mdx) first for key features such as filtering and update your configuration accordingly to achieve best performance before proceeding further.
+
+- `--network`, optional, can be one of Stellar's public networks, 'pubnet', or 'devnet'. Triggers horizon to automatically set configurations for remaining horizon settings and generate the correct core toml/cfg settings. If you only need Horizon to connect to one of those public Stellar networks, this will take care of all related configuration. **TODO - create this new flag and the functionality it performs to configure all aspects including toml, archive urls, etc for network 'pubnet' or 'testnet'**
+
+- If you want to connect Horizon to a different stellar network or override any of the defaults that horizon will make, the key configurations that can be set are: `--history-archive-urls`, `--captive-core-config-path`, `--network-pass-phrase`, `--captive-core-storage-path`, `--stellar-core-url`.
+
+- `--db-url`, required, specifies the Horizon database. It's value follows this format: `dbname=<pg_dbname> user=<pg_user_name> password=<pg_user_pwd> host=<host_address>`
+
+- `--log-level`, optional, can be one of 'info', 'error', 'debug'.
+
+- Horizon will create a sub-directory under the current working directory of the o/s process to store captive core runtime data files. Expect up to 10GB usage(**TODO - verify the max storage needs of core/buckets on disk?**). You can override this location with the optional `--captive-core-storage-path`, set to a directory on file system which captive core wiull store the runtime files.
+
+### Passing configurations to Horizon
+
+Depends on how Horizon was installed:
+
+- bare-metal
+  - non-package manager, we recommend using environment variables to pass configurations. There are many tools you can use to manage them, such as [direnv](http://direnv.net/) or [dotenv](https://github.com/bkeepers/dotenv).
+  - [package manager](./installing.mdx#package-manager), the provided `stellar-horizon-cmd` wrapper will import a configuration from `/etc/default/stellar-horizon`, hence, if you want to set configurations, edit the file at `/etc/default/stellar-horizon`.
+    <Alert>
+      This script invokes Horizon with the `stellar` user, so make sure that
+      permissions for the user are set up accordingly. The current working
+      directory should be writable for this user and the user should be able to
+      execute the `horizon` and `stellar-core` binaries; etc.
```

Comment:
Should this be "execute the `stellar-horizon`" instead since the name of the binary is stellar-horizon?

Reply:
It looks like the binary name is "horizon" when downloaded as prebuilt binaries or built locally but it is "stellar-horizon" when installed via package manager. This distinction becomes relevant when users copy and paste example commands from our docs as it may not work correctly depending on the installation method. Previously the documentation had an additional step of renaming the binary using `sudo cp horizon /usr/local/bin/stellar-horizon` I'm uncertain about the reasoning behind the difference in names and the steps required to unify 
but mentioning it here as a potential TODO item (?).